veocean-data-farming/faq.md

```diff
@@ -0,0 +1,121 @@
+---
+description: Frequently asked questions on veOCEAN, Data Farming, and our incentive programs.
+---
+# FAQ
```

Comment:
I don't think it makes sense to introduce another FAQ page within the docs. It would be better to add data farming as a section within the main FAQ section. If you feel like the main FAQ is hard to find you can bring it up to the top level in the navigation

Reply:
You could also remove the Discord link from the top nav (we have that on the side anyway) and add the FAQ link there

![image](https://user-images.githubusercontent.com/20739535/227456114-caf175f1-b8b0-4bda-9839-76b9e3a139aa.png)

Reply:
Agree with Jamie , add everything in the existing FAQ, move it on the first level (under `API Refrences` ) so it's visible and we can add it in the header instead of Discord . Don't necessarily focus more on the structure since we will come with a proposal to restructure the docs soon.

Reply:
1. FAQs merged. Done.
```You could also remove the Discord link from the top nav (we have that on the side anyway) and add the FAQ link there```
2. I do not have access to gitbook so I can't change the navbar on the project. Resolving the conversation for now and will update the navbar elsewhere. Let's take it in a different task. :scroll:

Reply:
I'll adjust the link at the top in gitbook. Can you add the FAQ to the high level on the side nav like Mihai suggested? You can do that in `./SUMMARY.md`